its/pom.xml

```diff
@@ -164,8 +164,8 @@
           <appArgLine>-eclipse.keyring target/keyring -eclipse.password secure-storage-password.txt -pluginCustomization
             ${project.basedir}/plugin_customization.ini</appArgLine>
           
-          <!-- Kill test JVM if tests take more than 45 minutes (2700 seconds) to finish -->
-          <forkedProcessTimeoutInSeconds>2700</forkedProcessTimeoutInSeconds>
+          <!-- Kill test JVM if tests take more than 60 minutes (3600 seconds) to finish -->
+          <forkedProcessTimeoutInSeconds>3600</forkedProcessTimeoutInSeconds>
```

Comment:
We hit the mark too often, that is why I increased it. I've spoken to RE and we ended up with [THIS](https://sonarsource.atlassian.net/browse/BUILD-4406) follow-up ticket, that I'm going to tackle after this one is merged - build times should decrease.

Reply:
Maybe we should rely only on Cirrus timeout, and not on this Maven feature at all? This look like a remain of the past, AFAIK no other project is using surefire timeout.